### DISCUSSION TITLE
Give output of only sunset results, not sunrise as well

### DISCUSSION THREAD

**Discussion by hidp123** [skyfielders/python-skyfield#766] [Q&A] [answered]
Using the `risings_and_settings()` routine I can get results for sunrise/moonrise and sunset/moonset. Is there a way I can output just the sunset/moonset results.

**Comment by Bernmeister** [comment]
Unless I'm missing something, when iterating over the events list (comprising a run of `0` and `1`) from calling `almanac.find_discrete()`, skip over each `1`'?

**Reply by lulunac27a** [reply, marked as answer]
Use `almanac.find_settings()` to find the setting time of an object (sun and moon) from the start date/time to end date/time.
For example: `almanac.find_settings(observer_location, eph['Sun'], start_time, end_time)`
